packages/eas-cli/src/graphql/queries/EnvironmentSecretsQuery.ts

```diff
@@ -60,4 +61,18 @@ export const EnvironmentSecretsQuery = {
 
     return data.app?.byFullName.environmentSecrets ?? [];
   },
+  async all(
```

Comment:
```suggestion
  async allAsync(
```
We use suffix Async to all async functions